recipe/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tsai" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d54d88ca40d27d29e96adc98cb9641c848411caedaf517a93dce3b64c4d37915
+
+build:
+  number: 0
+  skip: True  #[py<37 or py>310 or s390x or ppc64le]
+  entry_points:
+    - nb2py=tsai.export:nb2py
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+    - setuptools
+  run:
+    - fastai >=2.7.12
+    - imbalanced-learn >=0.8.0
+    - psutil >=5.4.8
+    - python
+    - pyts >=0.12.0
+    - pytorch >=1.7,<2.1
+    - ipykernel
+    - numba >=0.48.0
```

Comment:
`numba` was removed by the maintainers on Mar 2, 2022 
 https://github.com/timeseriesAI/tsai/commit/f083f2a27b09f007e1759c704e248908b5119de3#diff-f247846b0ab6c196467cd7e8e41027c7f27eb2b74b2e9a33d54c59a6fe9f00b0